### ISSUE TITLE
`rebar.config.script` has permissions `660`

### ISSUE THREAD

**Issue by kivra-pauoli** [g-andrade/tls_certificate_check#52]
👋 

The distributed `rebar.config.script` has permissions 660, instead of expected (?) 664.

This causes issues in CI:

```
/usr/bin/tar: _build/default/lib/tls_certificate_check/rebar.config.script: Cannot open: Permission denied
/usr/bin/tar: Exiting with failure status due to previous errors
Warning: Failed to save: "/usr/bin/tar" failed with error: The process '/usr/bin/tar' failed with exit code 2
```

I'm not really sure why it:

1. needs those permissions, so assume you do?
2. doesn't fail in other circumstances, but so specifically when GHA's `actions/checkout` is trying to create a cache.

Thanks for the lib. and taking the time to look at this.

**Comment by KristianAsp** [comment]
Were you able to fix this on your end @kivra-pauoli ? I've bumped into the same issue and struggling to get around it. 

Cheers

**Comment by kivra-pauoli** [comment]
No fix yet. We could run a script to change permissions after the file is downloaded, but this is no real fix 😄

**Comment by g-andrade** [comment]
Hi :wave: !

Thanks both for reporting the issue, I fixed it in 298d83e and a new package version will be pushed to Hex.pm.

There was no reason for the missing permissions, likely it was just a paranoid adjustment to local permissions at some directory above that leaked through. And I learned that Hex.pm packages preserve file mods...

**Comment by g-andrade** [comment]
Hex.pm diff doesn't show the fixed permissions (!)
https://diff.hex.pm/diff/tls_certificate_check/1.24.0..1.25.0

But I did confirm the fix after upgrading another library's own import of `tls_certificate_check`: `rebar.config.script` now has permissions 664.
